### ISSUE TITLE
[ICON REQUEST]: laminas/mezzio

### ISSUE THREAD

**Issue by pine3ree** [devicons/devicon#2234]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Laminas/Mezzio (community based successor of ZF3) is a very popular php framework

### Links and sources

https://getlaminas.org/
https://docs.mezzio.dev/
https://github.com/laminas
https://github.com/mezzio

#### Laminas logo on github 
<img src="https://avatars.githubusercontent.com/u/47865178?s=64&v=4" alt="Laminas" width="64" style="height:auto">

#### Mezzio logo on github 
Basically the same but in green
<img src="https://avatars.githubusercontent.com/u/58085692?s=64&v=4" alt="Mezzio" width="64" style="height:auto">

There are actually 4 related projects/libraries with the same logo but with different colors: https://docs.laminas.dev/
